contents/blog/secrets-of-posthog-query-performance.md

```diff
@@ -0,0 +1,152 @@
+---
+date: "2022-03-16" # :TODO
+title: "Secrets of PostHog query performance"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/launch-week-teaser.jpeg # :TODO:
```

Comment:
@lottiecoxon You mentioned you had something planned for this post - is the image already available?